### PR TITLE
Enable new feedback system.

### LINKED ISSUE #37025: [Experience] New feedback system coming in September
The learn platform has a new feedback system for reporting issues in the `dotnet/docs` and related repositories.

The new feedback system is integrated into the GitHub issue template system. The document block will be replaced with fields in a GH issue template form.

In general, the changes should be transparent for those reading our docs. For those customers that regularly participate in our repository, you'll see a new GH issue template, but you should only use it through the feedback system.

---
[Associated WorkItem - 157537](https://dev.azure.com/msft-skilling/Content/_workitems/edit/157537)

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #37025

Enable the new feedback system.

[Internal preview link](https://review.learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-12?branch=pr-en-us-37044)

You can try this on any page, but you'll need the PR branch, and the extra query string parameter `preview=open-source-feedback` to see the new experience.

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -126,6 +126,14 @@
             "featureFlags": [
                 "enable_uhf_ppe"
             ],
+            "open_source_feedback": {
```
Comment (homing1) [inline docfx.json:129]:
```suggestion
"open_source_feedback_contributorGuideUrl": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
"open_source_feedback_issueTemplateName": "customer-feedback",
"open_source_feedback_productName": ".NET",
"open_source_feedback_productDescription": "Free. Cross platform. Open source. A developer platform for building all your apps: web, mobile, desktop, gaming, IoT, and more. Supported on Windows, Linux, and macOS.",
"open_source_feedback_productLogoLightUrl": "https://learn.microsoft.com/media/logos/logo_net.svg",
"open_source_feedback_productLogoDarkUrl": "https://learn.microsoft.com/media/logos/logo_net.svg",
```

File: docfx.json [hunk 2]
```diff
@@ -126,6 +126,12 @@
             "featureFlags": [
                 "enable_uhf_ppe"
             ],
+            "contributorGuideUrl": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
```
Comment (homing1) [inline docfx.json:129]: Please add the `open_source_feedback_` prefix to the fields.

open_source_feedback_contributorGuideUrl
open_source_feedback_issueTemplateName
open_source_feedback_productName
open_source_feedback_productDescription
open_source_feedback_productLogoLightUrl
open_source_feedback_productLogoDarkUrl

Comment (BillWagner) [reply docfx.json:129]: Thanks @homing1 I mis-read your previous comment.

In the next commit, I'll also move this to the file meta data section.

File: docfx.json [hunk 3]
```diff
@@ -126,6 +126,12 @@
...
+            "issueTemplateName": "customer-feedback",
+            "productName": ".NET",
+            "productDescription": "Free. Cross platform. Open source. A developer platform for building all your apps: web, mobile, desktop, gaming, IoT, and more. Supported on Windows, Linux, and macOS.",
+            "productLogoLightUrl": "https://learn.microsoft.com/media/logos/logo_net.svg",
+            "productLogoDarkUrl": "https://learn.microsoft.com/media/logos/logo_net.svg",
             "feedback_system": "GitHub",
```
Comment (homing1) [inline docfx.json:135]:
```suggestion
"feedback_system": "OpenSource",
```

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

File: docfx.json [hunk 4]
```diff
@@ -162,12 +162,38 @@
         },
         "fileMetadata": {
             "feedback_system": {
+                "docs/**.md":  "OpenSource",
+                "docs/**.yml": "OpenSource",
```
Comment (IEvangelist) [inline docfx.json:166]:
```suggestion
                "docs/**.*":  "OpenSource",
```

File: docfx.json [hunk 5]
```diff
@@ -162,12 +162,38 @@
...
                 "docs/standard/design-guidelines/**/**.md": "None",
                 "docs/framework/data/adonet/**/**.md": "None",
                 "docs/framework/data/wcf/**/**.md": "None",
                 "docs/framework/ui-automation/**/**.md": "None",
                 "docs/framework/wcf/**/**.md": "None"
             },
+            "open_source_feedback_contributorGuideUrl": {
+                "docs/**/**.md":  "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
+                "docs/**/**.yml": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute"
```
Comment (IEvangelist) [inline docfx.json:175]:
```suggestion
                "docs/**/**.*":  "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute"
```

File: docfx.json [hunk 6]
```diff
@@ -162,12 +162,38 @@
...
+            },
+            "open_source_feedback_issueTemplateName": {
+                "docs/**.md":  "customer-feedback",
+                "docs/**.yml": "customer-feedback"
```
Comment (IEvangelist) [inline docfx.json:179]:
```suggestion
                "docs/**.*":  "customer-feedback"
```

File: docfx.json [hunk 7]
```diff
@@ -162,12 +162,38 @@
...
+            },
+            "open_source_feedback_productName": {
+                "docs/**.md":  ".NET",
+                "docs/**.yml": ".NET"
```
Comment (IEvangelist) [inline docfx.json:183]:
```suggestion
                "docs/**.*":  ".NET"
```

File: docfx.json [hunk 8]
```diff
@@ -162,12 +162,38 @@
...
+            },
+            "open_source_feedback_productDescription": {
+                "docs/**.md":  "Free. Cross platform. Open source. A developer platform for building all your apps.",
+                "docs/**.yml": "Free. Cross platform. Open source. A developer platform for building all your apps."
```
Comment (IEvangelist) [inline docfx.json:187]:
```suggestion
                "docs/**.*":  "Free. Cross platform. Open source. A developer platform for building all your apps."
```

File: docfx.json [hunk 9]
```diff
@@ -162,12 +162,38 @@
...
+            },
+            "open_source_feedback_productLogoLightUrl": {
+                "docs/**.md":  "https://learn.microsoft.com/media/logos/logo_net.svg",
+                "docs/**.yml": "https://learn.microsoft.com/media/logos/logo_net.svg"
```
Comment (IEvangelist) [inline docfx.json:191]:
```suggestion
                "docs/**.*":  "https://learn.microsoft.com/media/logos/logo_net.svg"
```

File: docfx.json [hunk 10]
```diff
@@ -162,12 +162,38 @@
...
+            },
+            "open_source_feedback_productLogoDarkUrl": {
+                "docs/**.md":  "https://learn.microsoft.com/media/logos/logo_net.svg",
+                "docs/**.yml": "https://learn.microsoft.com/media/logos/logo_net.svg"
```
Comment (IEvangelist) [inline docfx.json:195]:
```suggestion
                "docs/**.*":  "https://learn.microsoft.com/media/logos/logo_net.svg"
```

**Review by IEvangelist — COMMENTED** [review]
Simplify this configuration such that it's for any published file.

File: docfx.json [hunk 11]
```diff
@@ -162,12 +162,34 @@
...
                 "docs/framework/wcf/**/**.md": "None"
             },
+            "open_source_feedback_contributorGuideUrl": {
+                "docs/**/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute"
+            },
+            "open_source_feedback_issueUrl": {
+                "docs/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml"
+            },
+            "open_source_feedback_productName": {
+                "docs/**.*": ".NET"
```
Comment (IEvangelist) [inline docfx.json:179]:
```suggestion
                "docs/**.*": ".NET feedback"
```

File: docfx.json [hunk 12]
```diff
@@ -162,12 +162,34 @@
...
+            },
+            "open_source_feedback_productDescription": {
+                "docs/**.*": "Free. Cross platform. Open source. A developer platform for building all your apps."
```
Comment (IEvangelist) [inline docfx.json:182]:
```suggestion
                "docs/**.*": "Like the .NET product, our documentation is Open source. Provide feedback here."
```

File: docfx.json [hunk 13]
```diff
@@ -162,12 +162,34 @@
...
+                "docs/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml"
+            },
+            "open_source_feedback_productName": {
+                "docs/**.*": ".NET feedback"
+            },
+            "open_source_feedback_productDescription": {
+                "docs/**.*": "The .NET documentation is open source. Provide feedback here."
+            },
+            "open_source_feedback_issueTitle": {
+                "docs/**.*": "[Customer issue] - "
```
Comment (BillWagner) [inline docfx.json:185]: Let's leave the title blank:

```suggestion
                "docs/**.*": ""
```

**Comment by BillWagner** [comment]
closing and reopening for a fresh build.

**Review by gewarren — APPROVED** [review]

**Pull request merged**